jobs/JGLOBAL_MARINE_BMAT

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+export DATA="${DATAROOT}/${RUN}marinebmat_${cyc}"
+export DATA=${DATA:-${DATAROOT}/${RUN}marinebmat_${cyc}}
```

Comment:
Guessing the first one needs to be removed here?

Reply:
good catch